### PR TITLE
Register reader, writer, sink, and functions explicitly in QueryRepalyer

### PR DESCRIPTION
**Pull request by duanmeng**
Register reader, writer, sink, and functions explicitly in QueryRepalyer

### CODE REVIEW

**Review by xiaoxmeng — COMMENTED** [review]
@duanmeng LGTM % nit. Thanks!

File: velox/tool/trace/QueryReplayer.cpp [hunk 1]
```diff
@@ -20,15 +20,25 @@
...
 #include "velox/connectors/hive/storage_adapters/s3fs/RegisterS3FileSystem.h"
 #include "velox/core/PlanNode.h"
+#include "velox/dwio/dwrf/RegisterDwrfReader.h"
+#include "velox/dwio/dwrf/RegisterDwrfWriter.h"
+#include "velox/dwio/parquet/RegisterParquetReader.h"
+#include "velox/dwio/parquet/RegisterParquetWriter.h"
 #include "velox/exec/PartitionFunction.h"
 #include "velox/exec/QueryTraceUtil.h"
+#include "velox/expression/Expr.h"
+#include "velox/functions/prestosql/aggregates/RegisterAggregateFunctions.h"
```
Comment (xiaoxmeng) [inline velox/tool/trace/QueryReplayer.cpp:38]: Use macro around parquet things? thanks!

**Review by xiaoxmeng — APPROVED** [review]
@duanmeng LGTM. Thanks!

File: velox/tool/trace/QueryReplayer.cpp [hunk 2]
```diff
@@ -20,15 +20,29 @@
...
 #include "velox/connectors/hive/TableHandle.h"
 #include "velox/connectors/hive/storage_adapters/abfs/RegisterAbfsFileSystem.h"
 #include "velox/connectors/hive/storage_adapters/gcs/RegisterGCSFileSystem.h"
 #include "velox/connectors/hive/storage_adapters/hdfs/RegisterHdfsFileSystem.h"
 #include "velox/connectors/hive/storage_adapters/s3fs/RegisterS3FileSystem.h"
 #include "velox/core/PlanNode.h"
+#include "velox/dwio/dwrf/RegisterDwrfReader.h"
+#include "velox/dwio/dwrf/RegisterDwrfWriter.h"
+
+#ifdef VELOX_ENABLE_PARQUET
```
Comment (majetideepak) [inline velox/tool/trace/QueryReplayer.cpp:34]: This ifdef is not required. The Parquet headers have these ifdefs. We did this to keep the client API clean.

Comment (duanmeng) [reply velox/tool/trace/QueryReplayer.cpp:34]: @majetideepak Got it, thanks. Will remove this in the next PR. cc @xiaoxmeng

**Pull request closed without merging**